internal/core/command/v2/controller/http/command_test.go

```diff
@@ -113,6 +119,46 @@ func buildDeviceProfileResponse() responseDTO.DeviceProfileResponse {
 	return deviceResponse
 }
 
+func buildDeviceServiceResponse() responseDTO.DeviceServiceResponse {
+	service := dtos.DeviceService{
+		Name:        testDeviceServiceName,
+		BaseAddress: testBaseAddress,
+	}
+	return responseDTO.DeviceServiceResponse{
+		Service: service,
+	}
+}
+
+func buildReading() dtos.BaseReading {
+	id, _ := uuid.NewUUID()
+	baseReading := dtos.BaseReading{
+		Id:            id.String(),
+		DeviceName:    testDeviceName,
+		ResourceName:  testResourceName,
+		ProfileName:   testProfileName,
+		ValueType:     v2.ValueTypeUint16,
+		SimpleReading: dtos.SimpleReading{Value: "45"},
+	}
+	return baseReading
+}
+
+func buildEvent() dtos.Event {
+	id, _ := uuid.NewUUID()
+	reading := buildReading()
+	return dtos.Event{
+		Id:          id.String(),
+		DeviceName:  testDeviceName,
+		ProfileName: testProfileName,
+		Readings:    []dtos.BaseReading{reading},
+	}
+}
```

Comment:
Use new `NewEvent()` and `AddSimpleReading()` factory methods. If need then override Ids with your id.
```
event := NewEvent(testProfileName, testDeviceName)
event.AddSimpleReading(testResourceName, v2.ValueTypeUint16, , uint16(45))
event.Id = id.String()
event.Readings[0].Id = id.String()
return event
```

Reply:
Modified.